### PR TITLE
feat: Fix capital charge at 1bp  for USDC/DAI/ETH

### PR DESCRIPTION
**Pull request by nicholaspai**

### CODE REVIEW

**Review by cc7768 — APPROVED** [review]
lgtm

**Comment by nicholaspai** [comment]
Reminder to self. Need to update this modal also https://github.com/across-protocol/frontend-v2/blob/6d489f7e242731767e23be322c548358515f2922/src/views/Transactions/components/SpeedUpModal/SpeedUpModal.tsx#L71

File: api/_constants.ts [hunk 1]
```diff
@@ -8,14 +8,14 @@ export const disabledL1Tokens = [
 
 export const relayerFeeCapitalCostConfig = {
   ETH: {
-    lowerBound: ethers.utils.parseUnits("0.0003").toString(),
-    upperBound: ethers.utils.parseUnits("0.0006").toString(),
+    lowerBound: ethers.utils.parseUnits("0.0001").toString(),
```
Comment (mrice32) [inline api/_constants.ts:11]: Should we make this configurable via an env? That way they're synchronized across the API and FE?

Comment (mrice32) [reply api/_constants.ts:11]: Oh I forgot the FE calls the API instead of computing itself. But either way, might be better to use an ENV.

Comment (nicholaspai) [reply api/_constants.ts:11]: The FE reads from API we ended up removing the same constants file from the FE folder

File: src/utils/constants.ts [hunk 2]
```diff
@@ -636,7 +636,7 @@ export const wbtcLpCushion = process.env.REACT_APP_WBTC_LP_CUSHION || "0";
 export const daiLpCushion = process.env.REACT_APP_DAI_LP_CUSHION || "0";
 
 export const maxRelayFee = 0.25; // 25%
-export const minRelayFee = 0.0003; // 0.03%
+export const minRelayFee = 0.0001; // 0.03%
```
Comment (dohaki) [inline src/utils/constants.ts:639]: nit
```suggestion
export const minRelayFee = 0.0001; // 0.01%
```

**Review by dohaki — APPROVED** [review]
LGTM 👍 I left one nit

File: api/_constants.ts (hunk 1 above)
Comment (mrice32) [reply api/_constants.ts:11]: LGTM! But I do think that we might want to have this set via an env.

**Review by mrice32 — APPROVED** [review]

File: api/_constants.ts [hunk 3]
```diff
@@ -8,14 +8,14 @@ export const disabledL1Tokens = [
...
+    upperBound: ethers.utils.parseUnits("0.0001").toString(),
     cutoff: ethers.utils.parseUnits("750").toString(),
```
[the remark this answers is not in the document]
Comment (nicholaspai) [reply api/_constants.ts:13]: Nope not if they are the same value

**Pull request merged**